src/Hydra/Serializer/CollectionNormalizer.php

```diff
@@ -39,21 +39,16 @@ final class CollectionNormalizer implements NormalizerInterface, NormalizerAware
     use NormalizerAwareTrait;
 
     public const FORMAT = 'jsonld';
-    public const IRI_ONLY = 'iri_only';
 
     private $contextBuilder;
     private $resourceClassResolver;
     private $iriConverter;
-    private $defaultContext = [
```

Comment:
I suggest to keep the context, but default to the value of the attribute. It will allow to make it dynamic mechanism in the future as suggested by @teohhanhui.